examples/token_classification/peft_lora_ner.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import required libraries\n",
```

Comment:
Let's add a few sentences to the beginning before starting with the code. Something along the lines of: "In this notebook, we learn how to ...". You can also add the link to the course here.